src/main/java/seedu/ui/ExceptionTextUi.java

```diff
@@ -377,9 +374,9 @@ public static void invalidTwitterUsernameInput() {
     public static void invalidEmailInput() {
         String message = "The email id is not correctly formatted,\n"
                 + "Rules for email id :\n"
-                + "    * Lowercase letters only\n"
+                + "    * Lowercase and uppercase letters allowed\n"
                 + "    * Numbers, underscore, hyphen and dot allowed\n"
-                + "    * @ cannot be at the start or end";
+                + "    * @ cannot be at the start or end and only one allowed";
```

Comment:
I think the phrasing is slightly off, maybe it would be better as two separate lines?
 `Only one @ character allowed`
 `Email cannot start or end with @ character`

Reply:
Thanks for pointing out, will change as requested